pyart/graph/radardisplay.py

```diff
@@ -542,6 +550,15 @@ def plot_vpt(self, field, mask_tuple=None, vmin=None, vmax=None,
             sweeps from the plot.  False will include these rays in the plot.
             No rays are filtered when the antenna_transition attribute of the
             underlying radar is not present.
+        time_axis_flag : bool
+            True to plot the x-axis as time. False uses the default of index number.
+        dForm : str
```

Comment:
Add ",optional" to these and a message about how these parameters are ignored if time_axis_flag is set to False.